### PR TITLE
Create sz-cit.json

### PR DESCRIPTION
**Pull request by emmajclegg**
New list created as per #570.

A local company has advised that this register covers both NGOs and companies, although I have only been able to find information on company registrations so far.

### CODE REVIEW

File: lists/sz/sz-cit.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "coverage": [
+    "SZ"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company"
+  ],
+  "sector": [],
+  "code": "SZ-CIT",
+  "confirmed": false,
```
Comment (kd-ods) [inline lists/sz/sz-cit.json:19]:
```suggestion
  "confirmed": true,
```
This needs to be true in order for the information to be published on the org-id website.

File: lists/sz/sz-cit.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  "url": "https://www.gov.sz/index.php/ministries-departments/minnistry-of-commerce",
+  "description": {
+    "en": "Companies and non-governmental organisations (NGOs) are registered with the Government of Eswatini (Swaziland)'s Ministry of Commerce, Industry and Trade. They receive a registration ID, which does not change over time.\n\nA full list of services that the Ministry provides can be found at https://www.gov.sz/index.php/services-sp-831484471, and specific information on company registration here: https://www.gov.sz/index.php/departments-sp-1596706154?id=522. "
+  },
+  "coverage": [
+    "SZ"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company"
```
Comment (kd-ods) [inline lists/sz/sz-cit.json:15]:
```suggestion
    "company",
    "charity"
```
We use 'charity' as synonymous with NGO. Not ideal, I know.

File: lists/sz/sz-cit.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": {
+    "en": "Ministry of Commerce, Industry and Trade",
```
Comment (kd-ods) [inline lists/sz/sz-cit.json:3]:
```suggestion
    "en": "Ministry of Commerce, Industry and Trade, Eswatini", 
```

**Review by kd-ods — CHANGES_REQUESTED** [review]
Looking good, just a few things to edit.

**Comment by emmajclegg** [comment]
Thanks @kd-ods - those edits all make sense. I think I've accepted and committed them but anything else needed let me know

**Review by kd-ods — APPROVED** [review]

**Pull request merged**